analyzers/src/SonarAnalyzer.ShimLayer.CodeGeneration/SyntaxLightupGenerator.cs

```diff
@@ -982,6 +982,65 @@ private void GenerateSyntaxWrapperHelper(in GeneratorExecutionContext context, I
                     continue;
                 }
 
+                if (node.Name == "BaseNamespaceDeclarationSyntax")
```

Comment:
If I am not mistaken `SonarAnalyzer.ShimLayer.CodeGeneration` is only used to generate the wrappers so we should be fine with updating the roslyn version to be able to use the `BaseNamespaceDeclarationSyntax`. 
@costin-zaharia-sonarsource as you have more knowledge in this area, could you please confirm this?
```suggestion
                if (node.Name == nameof(BaseNamespaceDeclarationSyntax))
```